src/opm/io/eclipse/OutputStream.cpp

```diff
@@ -43,27 +43,37 @@
 namespace {
     namespace FileExtension
     {
+        std::string separate(const int   rptStep,
+                             const bool  formatted,
+                             const char* fmt_prefix,
```

Comment:
Any reason to not use `std::string`- avoiding the implicit conversion from `"..."`?

Reply:
No, probably not.  I would prefer `std::string_view` here to be honest, but we're not yet on C++17.  In this case I took the easy way out since I didn't actually need a `std::string` in this very special set of circumstances.